charts/backstage/Chart.yaml

```
@@ -38,4 +38,4 @@ sources:
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.9.2
+version: 1.9.3
```

let's keep it as it is since the fix are only related to the CI. We'll bump later in #178
```suggestion
version: 1.9.2
```

---

I think we need to update the version anyways, since the PR is changing the values schema though

---

yeah we can release the new schema in #178 as well